### ISSUE TITLE
Visualization issue on latest build with GNU

### ISSUE THREAD

**Issue by Djneu** [fastscape-lem/fastscapelib-fortran#49]
Hi,
I've been having an issue with the newest build where the VTK files do not show up in paraview on a GNU system. This seems to be an issue with the removal of "convert=big_endian" from the open function that writes the files, as setting this again fixes the issue.

From my understanding, it looks like cmakelists now sets the flag "-fconvert=big-endian", which should have been applied to the function and done the conversion without specifying it, but for some reason it seems this isn't occurring. The flags themselves seem to be included in the build, as currently with "-std=f2008" included adding "convert=big_endian" to the open function results in an error, but if it is removed I can again specify it like in previous builds. Because of this I'm not sure exactly why the convert flag isn't working, any ideas what the issue may be? 

Thank you for your time!

**Comment by benbovy** [comment]
Hi Derek,

Could you tell a bit more about the platform and versions (fastscapelib-fortran, gfortran...) that you use?

Have you tried setting the [GFORTRAN_CONVERT_UNIT](https://gcc.gnu.org/onlinedocs/gfortran/GFORTRAN_005fCONVERT_005fUNIT.html#GFORTRAN_005fCONVERT_005fUNIT) env variable?

**Comment by Djneu** [comment]
Yes, sorry this is for fastscalelib-fortran commit 48799f9 using gfortran GNU Fortran (GCC) 9.2.0.

I had not set the convert unit, and doing so fixed the issue. Is this something that should always be set or should the cmake flags work without it being set?

Either way, this is an easy fix, thank you for the solution!

**Comment by benbovy** [comment]
I'm not sure why it wouldn't work when building with gfortran via cmake. I've tried a while ago and didn't see any issue. 

Possible reasons would be that `CMAKE_Fortran_COMPILER_ID` doesn't match `Flang` or `GNU` (I'd be surprised since you mention that you use gfortran) or the `CMAKE_Fortran_FLAGS` is not properly updated in the CMakelists. It might also be worth looking at the build logs (verbose mode) to see if the flag is properly passed to the compiler.
